CHANGELOG.md

```diff
@@ -15,7 +21,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- markdown-link-check-disable -->
 
-[unreleased]: https://github.com/mineiros-io/terraform-google-gcr/compare/v0.0.1...HEAD
+[unreleased]: https://github.com/mineiros-io/terraform-google-gcr/compare/v0.0.2...HEAD
+[0.0.2]: https://github.com/mineiros-io/terraform-google-gcr/compare/v0.0.1...v0.0.2
```

Comment:
Is it expected that if I follow the link: "https://github.com/mineiros-io/terraform-google-gcr/compare/v0.0.1...v0.0.2"

I get a:

"
There isn’t anything to compare.
We couldn’t figure out how to compare these references, do they point to valid commits?
"

page ?

Reply:
yeah, because the release doesn't exist yet and will only be issued after this PR has been merged :)